source/security/firmware-update.rst

```diff
@@ -1,37 +1,372 @@
 Firmware Update
 ------------------
 
-|SPN| implements a robust, secure and power fail-safe firmware update mechanism. |SPN| image can be built to contain redundant boot components and depending on hardware assisted boot partition switch, to support fail-safe update.
+|SPN| implements a secure and power fail-safe firmware update mechanism.
 
-If firmware update is interrupted by a power failure and one boot partition is corrupted, |SPN| boots from the alternate partition to avoid bricking the platform. The firmware update authenticates the update image (a.k.a. capsule) before updating the firmware.
+  * |SPN| image is designed to contain redundant boot components and depends on hardware assisted boot partition switch to support fail-safe update.
 
-Firmware update feature is implemented as a |SPN| payload.
+  * Firmware update code will authenticates the capsule image before proceeding with firmware update.
 
-When firmware update is signaled, Stage 2 loads firmware update payload to start update flow. During the update, firmware update payload keeps a state machine to keep track of the update progress as follows:
+  * If firmware update is interrupted by a power failure and a boot partition is corrupted, |SPN| will boot from the alternate partition to avoid bricking the platform.
 
+  * Firmware update code maintains a state machine to keep track of the update progress.
 
-#. Boot from partition A
-#. If update flow is requested, load FWU payload
-#. FWU loads and verifies capsule data. If successful, update partition B
-#. Set partition B as 'active'
-#. Reboot
-#. Boot from partition B
-#. If update flow is in progress, load FWU payload
-#. FWU loads and verifies capsule data. If successful, update partition A
-#. Set partition A as 'active'
-#. End update flow state machine and reboot
+|SPN| supports independent update of BIOS and CSME regions of IFWI. Update of external configuration data is also supported.

```

Comment:
An overview of the high level flow is provided below

Reply:
Done